server/services/netatmo/lib/update/netatmo.updateWeatherStation.js

```diff
@@ -176,7 +176,7 @@ async function updateWeatherStation(key, device, deviceSelector) {
 
               try {
                 feature = await getDeviceFeatureBySelector(deviceModule, `${moduleSelector}-windstrength`);
-                if (feature.last_value !== windStrengthValue) {
+                if (parseFloat(feature.last_value) !== parseFloat(windStrengthValue)) {
```

Comment:
Any way to change this equality test somewhere else.
I see many many usage, you changed all. If you had a single method, you only need to change this single method to impact all cases.